telematic_system/telematic_historical_data_processing/Dockerfile

```diff
@@ -29,10 +29,7 @@ RUN chmod +x /ws/src/rosbag2_processing_service/docker/*
 RUN sudo bash /ws/src/rosbag2_processing_service/docker/checkout.sh
 RUN sudo bash /ws/src/rosbag2_processing_service/docker/install.sh
 
-COPY rosbag2_processing_service/docker/entrypoint.sh entrypoint.sh
-RUN chmod +x entrypoint.sh
-
 
 RUN echo "source /ws/install/setup.bash" >> ~/.bashrc
 
-CMD ["bash"]
\ No newline at end of file
+CMD ["ros2 run rosbag2_processing_service main"]
```

Comment:
Do we need this is command is already specified in the yml file?

Reply:
This is just to give an entrypoint for anyone using the dockerfile directly, without docker-compose